Week5/solution/Week5_Solution.ipynb

```diff
@@ -0,0 +1,159 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cbcc206c",
+   "metadata": {},
+   "source": [
+    "### INSERTING RECORDS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "1439aabd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def insertFunct():\n",
+    "    import csv    #importing csv functions\n",
+    "    fields=['sid','sname','city','contact']\n",
+    "    rows=[[101,'Om','Bardoli',9999990000],\n",
```

Comment:
Can we insert record by creating tuple instead of list as you write in this code?

Reply:
yes we can put tuples for each records in  a list

Reply:
Can you do code for that?

Reply:
We can code it with tuple like this-
```
rows=[(101,'Om','Bardoli',9999990000),
      (102,'Sai','Surat',9998888000),
      (103,'Ram','Navsari',9991110000),
      (104,'Vatsal','Vyara',9999666000),
      (105,'Harsh','Bardoli',8889900222)]
csvwriter.writerows(rows)
```